src/Documentation/Installation/Prerequisites.md

```diff
@@ -5,13 +5,19 @@ title: Prerequisites
 
 # Prerequisites
 
-Orleans is a set of .NET libraries. In order to use Orleans, you need [.NET Framework](http://www.microsoft.com/net) 4.5.1 or higher and a copy of [Visual Studio](https://www.visualstudio.com) 2015 or higher. Note that the Express versions of Visual Studio do not support extension packages, but you can use Orleans by adding references to the NuGet packages directly.
+Orleans is a set of .NET libraries delivered via [NuGet packages](NuGets.md).
+In order to use Orleans, you need [.NET Framework](http://dot.net) 4.6.1 (since 1.5.0, 4.5.1 for prior versions) or higher and a copy of [Visual Studio](https://www.visualstudio.com) 2015 or higher.
+Note that the Express versions of Visual Studio do not support extension packages, but you can use Orleans by adding references to the NuGet packages directly.
 
-In production, Orleans requires persistent storage. The following technologies are supported (only need one of those):
+In production, Orleans requires persistent storage for reliable cluster membership.
```

Comment:
I struggled with this a bit. The way I see it is the list enumerates all membership table storage options while SF is a different case - a deployment solution. For example, one could deploy Orleans as a stateless SF service with Azure Table or SQL as the membership store. That's why I felt simply adding SF to the list would be inaccurate.